app/src/main/java/com/edu/movie/screen/trending/MoviesTrendingFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.edu.movie.screen.trending
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import com.edu.movie.R
+import com.edu.movie.utils.TrendingMoviesParams
+
+class MoviesTrendingFragment : Fragment() {
+    private var trendingMoviesParams: TrendingMoviesParams? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            trendingMoviesParams = it.getSerializable(ARGUMENT_TRENDING) as TrendingMoviesParams?
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_movies_trending, container, false)
+    }
+
+    companion object {
+        private const val ARGUMENT_TRENDING = "ARGUMENT_TRENDING"
+
+        @JvmStatic
+        fun newInstance(param: TrendingMoviesParams) = MoviesTrendingFragment().apply {
+            arguments = bundleOf(Pair(ARGUMENT_TRENDING, param))
```

Comment:
Pair trong kotlin có thể viết dưới dạng KEY to Value
```suggestion
            arguments = bundleOf(ARGUMENT_TRENDING to  param)
```